From af65374f6ddfa46a5a55a9c6e2e82d209a419975 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Fri, 13 Jan 2017 17:58:14 +0800
Subject: [PATCH 1628/5242] MLK-13756 ARM: dts: imx6sll-evk-btwifi: change the
 pad setting for sd3

commit  6e898b013880a9b9a5e12e5991f10366cb7b7d83 from
https://source.codeaurora.org/external/imx/linux-imx.git

The pad setting suggested by HW team affects the normal function
of sdio wifi. This patch changes the pad setting for sd3.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Tiberiu Breana <andrei-tiberiu.breana@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll-evk-btwifi.dts |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll-evk-btwifi.dts b/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
index 3cfa327..db30b1f 100644
--- a/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
+++ b/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
@@ -45,6 +45,13 @@
 	imx6sll-evk-murata-v2 {
 		pinctrl_wifi: wifigrp {
 			fsl,pins = <
+				MX6SLL_PAD_SD3_CMD__SD3_CMD     0x170b9
+				MX6SLL_PAD_SD3_CLK__SD3_CLK     0x130b9
+				MX6SLL_PAD_SD3_DATA0__SD3_DATA0 0x170b9
+				MX6SLL_PAD_SD3_DATA1__SD3_DATA1 0x170b9
+				MX6SLL_PAD_SD3_DATA2__SD3_DATA2 0x170b9
+				MX6SLL_PAD_SD3_DATA3__SD3_DATA3 0x170b9
+
 				MX6SLL_PAD_KEY_COL0__GPIO3_IO24 0x17059 /* WL_REG_ON */
 				MX6SLL_PAD_KEY_COL1__GPIO3_IO26 0x17059 /* WL_HOST_WAKE */
 			>;
@@ -67,7 +74,7 @@
 
 &usdhc3 {
 	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_usdhc3_100mhz &pinctrl_wifi>;
+	pinctrl-0 = <&pinctrl_wifi>;
 	no-1-8-v;
 	non-removable;
 	cd-post;
-- 
1.7.9.5

